tools/importer-rest-api-specs/components/parser/constants/interface.go

```diff
@@ -145,13 +159,38 @@ func parseConstantExtensionFromExtension(field spec.Extensions) (*constantExtens
 	}
 
 	var enumName *string
+	var valuesToDisplayNames *map[interface{}]string
 	for k, v := range enumDetails {
 		// presume inconsistencies in the data
 		if strings.EqualFold(k, "name") {
 			normalizedEnumName := cleanup.NormalizeName(v.(string))
 			enumName = &normalizedEnumName
 		}
 
+		if strings.EqualFold(k, "values") {
+			items := v.([]interface{})
+			displayNameOverrides := make(map[interface{}]string)
+			for _, itemRaw := range items {
+				item := itemRaw.(map[string]interface{})
```

Comment:
Do we want to check that this isn't nil or is it fine?

Reply:
The extension is defined in this manner, and everything we're importing matches it - so I think this is fine as-is for now. If this is an issue we can fix it at that point, but that's likely going to be bad data - so would want resolving upstream anyway - as such this is fine 👍